2d/hydrostatic_column/mpm_usf_1.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Rename to `mpm_usf.json` no need to have multiple files for velocity update and not. Just one of these is sufficient.